components/zoho_crm/zoho_crm.app.mjs

```diff
@@ -35,13 +97,33 @@ module.exports = {
         headers,
       };
     },
+    _getAxiosParams(opts) {
+      console.log({
+        ...opts,
+        url: `${this.$auth.api_domain}/crm/v2${opts.path}`,
+        headers: {
+          "Authorization": `Zoho-oauthtoken ${this.$auth.oauth_access_token}`,
+          "User-Agent": "@PipedreamHQ/pipedream v0.1",
+        },
+        path: undefined,
+      });
+      return {
+        ...opts,
+        url: `https://www.zohoapis.com/crm/v2.1${opts.path}`,
+        headers: {
+          "Authorization": `Zoho-oauthtoken ${this.$auth.oauth_access_token}`,
+          "User-Agent": "@PipedreamHQ/pipedream v0.1",
+        },
+        path: undefined,
+      };
+    },
     async genericApiGetCall(url, params = {}) {
       const baseRequestConfig = this._makeRequestConfig();
       const requestConfig = {
         ...baseRequestConfig,
         params,
       };
-      const { data } = await axios.get(url, requestConfig);
+      const { data } = await axios(this, url, requestConfig);
```

Comment:
Should this be `const data =` since axios is replaced with `@pipedream/platform` axios?